src/manager.cpp

```diff
@@ -116,6 +116,7 @@ void Manager::SetTimerToDetectSVPDOnDbus()
             m_worker->collectFrusFromJson();
             m_interface->set_property("CollectionStatus",
                                       std::string("Completed"));
+            m_worker->backupAndRestore();
```

Comment:
Did not understand this. Why manager is calling via worker?

Reply:
Removed this statement, only including BackupRestore API's implementation in this PR.